### PR TITLE
Fix: Lighthouse issues & PageSpeed score

### PR DESCRIPTION
**Pull request by cp-mansi-d**

### CODE REVIEW

File: pages/_document.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import Document, { Html, Head, Main, NextScript } from "next/document";
```
Comment (cp-sumi-k) [inline pages/_document.js:1]: What is purpose of it?

File: pages/index.js [hunk 2]
```diff
@@ -90,12 +90,16 @@ export default function Home({ posts, status }) {
                           } `
                     }`}
                   >
-                    <Link href={"/post/" + post.slug}>
+                    <Link
+                      href={"/post/" + post.slug}
+                      aria-label={"Read more about" + post.title}
```
Comment (cp-sumi-k) [inline pages/index.js:95]: let's put space after `about`

File: pages/post/[slug].js [hunk 3]
```diff
@@ -207,14 +211,14 @@ export default function Post({ postData, status }) {
...
-                <div className="container flex flex-col xl:flex-row space-y-20 xl:space-y-0 xl:space-x-20 2xl:mx-10 rounded-3xl text-[1.125rem] font-source-serifpro">
+                <div className="container flex flex-col xl:flex-row space-y-20 xl:space-y-0 xl:space-x-20 2xl:mx-10 rounded-3xl text-[1.125rem]">
                   {indexContent != null ? (
                     <div className="xl:sticky top-24 w-auto h-60 xl:h-fit w-[100%] xl:w-[30%] border border-1 border-black-900 rounded-[12px] overflow-y-auto">
                       <div className="rounded-t-[12px] bg-gray-100 py-5 pl-4">
                         Table of contents
                       </div>
-                      <div className="pl-5 pr-8 text-gray-700 font-normal tracking-[-0.003em] leading-relaxed">
-                        <div className="mt-4 text-[1rem] md:text-[1.125rem] list-none ">
+                      <div className="pl-5 pr-8 text-[#292929fa] font-normal tracking-[0.02em] leading-relaxed">
```
Comment (cp-sumi-k) [inline pages/post/[slug].js:220]: Will this color code work?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: pages/_document.js (hunk 1 above)
Comment (cp-mansi-d) [reply pages/_document.js:1]: Lighthouse returns this error : The <html> element does not have a [lang] attribute
, so I tried to add lang in inner html tag in _app.js file but it generates new error, so I found this solution and this solution works properly.

File: pages/post/[slug].js (hunk 3 above)
Comment (cp-mansi-d) [reply pages/post/[slug].js:220]: yes, this is work.

Comment (cp-sumi-k) [reply pages/post/[slug].js:220]: I think this will not work, Check once

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**